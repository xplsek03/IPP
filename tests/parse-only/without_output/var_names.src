.IPPcode19
DEFVAR GF@_-$&%*abcdefghijklmnopqrstuvwxyz123456789
DEFVAR GF@ceskeznaky



