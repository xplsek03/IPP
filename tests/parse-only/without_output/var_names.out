<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode19">
 <instruction order="1" opcode="DEFVAR">
  <arg1 type="var">GF@_-$&amp;%*abcdefghijklmnopqrstuvwxyz123456789</arg1>
 </instruction>
 <instruction order="2" opcode="DEFVAR">
  <arg1 type="var">GF@ceskeznaky</arg1>
 </instruction>
</program>
